snippets/json-cleaner.md

```diff
@@ -0,0 +1,20 @@
+### Clean Json objects
```

Comment:
This file is on a different PR, please remove it from this one and we will figure it out in #125.